app/code/Magento/Catalog/Model/Product/Attribute/Source/StockStatus.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Catalog\Model\Product\Attribute\Source;
+
+use Magento\Eav\Model\Entity\Attribute\Source\AbstractSource;
+use Magento\Eav\Model\Entity\Attribute\Source\SourceInterface;
+use Magento\Framework\Data\OptionSourceInterface;
+
+/**
+ * Product stock status functionality model
+ */
+class StockStatus extends AbstractSource implements SourceInterface, OptionSourceInterface
+{
+    /**
+     * Stock status value for product in stock
+     */
+    const IN_STOCK = 1;
+
+    /**
+     * Stock status value for out of stock product
+     */
+    const OUT_OF_STOCK = 0;
+
+    /**
+     * Retrieve option array
+     *
+     * @return string[]
+     */
+    public static function getOptionArray() : array
```

Comment:
Static methods should not be used according to [Magento coding statndatds](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#2-class-design)

Reply:
changes was provided

Reply:
@XxXgeoXxX I don't see these changes... Can you also squash those 18 commits into a single after the update? Thanks